client/src/components/Ratings.jsx

```diff
@@ -7,20 +7,32 @@ import { token } from '/config.js';
 
 function Ratings ({product_Id, productName}) {
   const [productReviews, setProductReviews] = useState({});
+  const [sort, setSort] = useState('relevant');
+  const [reviews, setReviews] = useState([]);
+  const [starsFilter, setStarsFilter] = useState([]);
+  const [revert, setRevert] = useState(false);
+  const [starsClicked, setStarsClicked] = useState({
+    '1': false,
+    '2': false,
+    '3': false,
+    '4': false,
+    '5': false
+  });
+
 
   useEffect(() => {
-    const url = `http://127.0.0.1:3000/reviews?product_id=${product_Id}`;
+    const url = `http://127.0.0.1:3000/reviews`;
+    const params = {product_id: product_Id, count: 50, sort : sort};
     const fetchReviews = async () => {
       const getReviews = await Axios.get(url, {
-        headers: {
-          'Authorization': token
-        }
+        params: params
```

Comment:
Nice! Liking the use of params. Similar could be done within your `useEffect` on Line 40. Also, consider removing the `headers` on Lines 45-47. These should be handled by the Server. I'll approve & merge w/o these changes. Not indispensable.

Reply:
i took out the authorization but left the url for now so that the page works. once i do your code review and all files are merged ill switch out the url var.